server/controllers/admin.py

```diff
@@ -440,7 +467,7 @@ def templates(cid, aid):
         if files:
             templates = {}
             for template in files:
-                templates[template.filename] = str(template.read(), 'latin1')
+                templates[template.filename] = str(template.read(), 'utf-8')
```

Comment:
I found the commit: 22b7f32

Seems like we should store files as binary data rather than as a string, but since we store files as json we have to encode it somehow. I'll look how ok-client does it